### ISSUE TITLE
‍diffForHumans: yesterday/tomorrow 

### ISSUE THREAD

**Issue by salarmehr** [briannesbitt/Carbon#625]
It's far better that diffForHumans return tomorrow/yesterday instead of '1 day after/ago'. Some languages (e.g. Persian) also have single word for `2 day after/before`.

**Comment by Oxicode** [comment]
+1

**Comment by kylekatarnls** [comment]
I would be a breaking change for everyone that expect to keep having the "1 day after/ago" result, and it would mean translate "tomorrow/yesterday" in 63 languages.

So I recommend to use a custom implementation (extend or macro):

```php
public function customDiffForHumans(self $other = null, $absolute = false, $short = false, $parts = 1)
{
  $days = $this->diffInDays($other, $absolute);
  if ($days === 1) {
    return "tomorrow"; // You're on you own for translations
  }
  if ($days === -1) {
    return "yesterday";
  }

  return $this->diffForHumans($other, $absolute, $short, $parts);
}
```

**Comment by kylekatarnls** [comment]
And as we can notice, tomorrow/yesterday can be relevant only if `$absolute` is false.

**Comment by kylekatarnls** [comment]
This issue and #626 could be handled in a common way.

The default behavior must keep being what it's now for backward compatibility.

Settings method or options argument (binary values with pipes) could enable the following features:
- "Just now" ("now" translation key) if 0 seconds. (Keep 1 second if the "now" translation key is not available in the current locale)
- "yesterday/tomorrow" translation key if 1 or -1 day. (Keep 1 day after/before if translation keys not available)
- "avant-hier/après-demain" ("day_before_yesterday, day_after_tomorrow translation keys), French, Persian and all languages with words for 2 and -2 day (should be translated only if those words are common in those languages)

**Comment by kylekatarnls** [comment]
Will be fixed via #1240
